examples/tutorial/lazy-loading/README.md

```diff
@@ -0,0 +1,20 @@
+# Lazy Loading
```

Comment:
Better to choose a title that can capture users' mind about "synchronization".
To me,  'lazy' here is not that important, it's just a implementation detail.
how about 'synced-loading'?